README.md

```diff
@@ -158,6 +158,23 @@ The environment variables used by the frontend are also checked for type safety.
 3. Import it in the file where it's used with `import { env } from "@/env";` and use it like `env.MY_ENV_VAR`. At this point, your environment variable should be working locally.
 4. To ensure it also works in CircleCI and Silta, also add it to`.circleci/config.yml` and `silta-next.yml`.
 
+### Testing with Cypress
+
+The template includes example tests to be run with Cypress
```

Comment:
Maybe make it clearer here that the tests currently only work locally and do not run in CI.